Assets/Scripts/GUI/KeyboardPopUpWindow.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright 2022 The Tilt Brush Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.IO;
+
+namespace TiltBrush
+{
+    public class KeyboardPopUpWindow : OptionsPopUpWindow
+    {
+        private KeyboardUI m_KeyboardUI;
+        private const string m_validFilenameCharacters = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789._-";
```

Comment:
This doesn't appear to be used - is it meant to be?

Reply:
Good spot. I'll remove it

Reply:
See 420af0317d5c12f95c4233b71157f6c8ce6af3e9